src/tools/scripts/nettoyageUserCreationError.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../utils');
+const { DBRef } = require('mongodb');
+
+const getStructures = async db => await db.collection('structures').find({
+  userCreated: true, userCreationError: true
+});
+
+const getConseillers = async db => await db.collection('conseillers').find({
+  userCreated: true, userCreationError: true
+});
+
+const getUserByEntity = db => async entity => await db.collection('users').findOne({
+  'entity': entity
```

Comment:
peut etre spécifier le roles $in (structure ou candidat/conseiller) car _id est unique par collection

Reply:
tu peux cibler direct avec entity.$id